hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -280,6 +288,13 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
       openKeysError = true;
     }
 
+    if ([
+      decommissionInfoCount].some(
+        (data) => data === undefined
+      )) {
```

Comment:
Nit:
```suggestion
    if ([decommissionInfoCount].some(
        (data) => data === undefined
      )) {
```

Reply:
Done the changes in latest commit https://github.com/apache/ozone/commit/3e08fb7b48adea2a78e0f77408427da00db453bf and done testing with local cluster data.